### PR TITLE
ebcdic conversion fixes

### PR DESCRIPTION
**Pull request by HarithaIBM**
Fix for https://github.com/ZOSOpenTools/unzipport/issues/13

### CODE REVIEW

File: buildenv [hunk 1]
```diff
@@ -9,6 +9,10 @@ export ZOPEN_CONFIGURE_OPTS="."
 export ZOPEN_INSTALL="cmake"
 export ZOPEN_INSTALL_OPTS="--install . --prefix \$ZOPEN_INSTALL_DIR/"
 export ZOPEN_CHECK="${PWD}/unzip-tests/tests.sh"
+export ZOPEN_CPPFLAGS="-DNSIG=42 -D_XOPEN_SOURCE=600 -D_ALL_SOURCE -D_OPEN_SYS_FILE_EXT=1 -D_AE_BIMODAL=1 -D_ENHANCED_ASCII_EXT=0xFFFFFFFF"
+export ZOPEN_CFLAGS="-fzos-le-char-mode=ascii -std=gnu11 -mnocsect -fno-short-enums -fno-builtin-malloc -fno-builtin-calloc -fno-builtin-realloc -m64 -mzos-target=zosv2r4"
+export ZOPEN_CXXFLAGS="-fzos-le-char-mode=ascii -mnocsect -fno-short-enums -fno-builtin-malloc -fno-builtin-calloc -fno-builtin-realloc -m64 -mzos-target=zosv2r4"
+export ZOPEN_LDFLAGS="-Wl,-bedit=no"
```
Comment (IgorTodorovskiIBM) [inline buildenv]: Are these necessary? They should be coming from comp_clang

File: patches/extract.c.patch [hunk 2]
```diff
@@ -12,3 +20,12 @@ index 1acd769..0775a70 100644
  #endif
  #ifdef DOS_FLX_NLM_OS2_W32
  #if (defined(__HIGHC__) && !defined(FLEXOS))
+@@ -1727,7 +1729,7 @@ static int extract_or_test_member(__G)    /* return PK-type error code */
+             while ((b = NEXTBYTE) != EOF) {
+                 *G.outptr++ = (uch)b;
+                 if (++G.outcnt == wsize) {
+-                    error = flush(__G__ redirSlide, G.outcnt, 0);
```
Comment (IgorTodorovskiIBM) [inline patches/extract.c.patch:19]: Any changes in this file?

File: buildenv (hunk 1 above)
Comment (HarithaIBM) [reply buildenv]: Right, i dont need these now. I have removed it.

File: patches/extract.c.patch (hunk 2 above)
Comment (HarithaIBM) [reply patches/extract.c.patch:19]: Corrected indentation of this line.

**Comment by HarithaIBM** [comment]
Hi @IgorTodorovskiIBM I have addressed all the comments. Thank you.

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**